### DISCUSSION TITLE
CSS twks v4.3.5 - Something wrong with urlbar after a full screen video. maybe not a bug. don't know.

### DISCUSSION THREAD

**Discussion by fr-cerber** [Aris-t2/CustomCSSforFx#587] [General]
First of all,
I would like to say thank you for this work done over the years to allow further configuration of the firefox environment.

I have a possible bug with
Custom CSS tweaks for Firefox v4.3.5
Environment : Windows 10
Firefox version : 112.0 (64 bits)
Did I have the bug before? No or maybe … explanation after
Concerned file : userchrome.css
Concerned code
```
TABS BELOW NAVIGATION AND BOOKMARKS TOOLBARS / TABS NOT ON TOP
/* Windows/Linux */
/* @import "./css/tabs/tabs_below_navigation_toolbar.css"; /**/
```

Explanation of the bug:
In my previous version of css tweaks I had activated the following line:
```
@import "./css/tabs/tabs_below_navigation_toolbar_alt.css"; /**/
```
But I did not use multiple lines for bookmarks (No bug to report so far)

When switching to the new version: 4.3.5
I noticed that this row was only for those that have multiple rows.
So as I do not have multiple lines, I activated the following line:
`@import "./css/tabs/tabs_below_navigation_toolbar.css"; /**/`
Except that in doing so I have a bug:
Here is my interface each time firefox is launched.
![issue1](https://user-images.githubusercontent.com/130781364/232102294-e55fbb8e-3b3a-4226-8834-0280c503fe71.jpg)
No apparent visual bugs

But every time I launch a video in full screen here is my interface when I leave full screen.
![issue2](https://user-images.githubusercontent.com/130781364/232102361-fb34dc42-9857-4ea1-9923-f7d5aba28305.jpg)

The address bar takes up all the space and some of my buttons are stored in a folder of extra buttons.

So I went back to the other setting:
`@import "./css/tabs/tabs_below_navigation_toolbar_alt.css"; /**/`
and I no longer have this interface bug. 
But I still specify that I don't use multi-lined tabs and I tried removing this :

> /* [!] create additional toolbar for 'bookmarks toolbar items' on navigation toolbar */

And using the config :
`@import "./css/tabs/tabs_below_navigation_toolbar.css"; /**/`
to see if it could be taken into account in the bug, but it does not work either

as I don't know if it's a bug or a bad setting, I didn't want to open a bug report.
I hope what I am presenting is clear, if it is not a bug, please explain to me where I am making a configuration error. I can provide the userchrome file if needed.

thanks

**Comment by Aris-t2** [comment]
Does this also happen with default toolbar configuration?

Try to reset it in customizing mode and place the "searchbar" after "location bar" without "reload/strop buttons" between them.

**Comment by fr-cerber** [comment]
if u mean using firefox customisation mode to change the position of the reload button, yeah I tried that too.. I tried a lot of things. 
When i switched back to my old userchrome and saw that on the old it was working fine, i looked all over the new userchrome and saw that i had changed a line from the previous version. The one I mentioned above. 
I've looked everywhere for what could be causing this "bug" and this is the only line that seems to be involved.
Configuring the new userchrome by putting the same parameters as those of the previous version fixed the problem.

Maybe there is another change in the userchrome file that conflicts with this line, but I don't know what.
I can provide both userchrome files for comparison if you want.

**Comment by Aris-t2** [comment]
Is this issue still present. I'm still not able to reproduce it.
Fx 117 + this projects latest files.

**Comment by Aris-t2** [comment]
Closing this for now.
